### DISCUSSION TITLE
Access Denied on S3 Proxy

### DISCUSSION THREAD

**Discussion by rehatkathuria** [mastodon/mastodon#20104] [Q&A]
Profile: https://mn.reh.at/@toot

Avatar: https://assets.reh.at/accounts/avatars/109/302/605/019/088/032/original/37840b11f52cb919.jpg

Actual S3 URL: https://tootable.s3.amazonaws.com/accounts/avatars/109/302/605/019/088/032/original/37840b11f52cb919.jpg

Anyone know why this might be happening?

**Comment by tuetenk0pp** [comment]
I think I might have had a similar issue. Have a look at #19739.

**Reply by rehatkathuria** [reply]
Still seeing issues after following your suggestions!

**Reply by samuelbjohnson** [reply]
I'm seeing a similar issue. The suggestions in the above ticket did not help.
